### PR TITLE
#181 HackAUBG awards section

### PR DESCRIPTION
**Pull request by savinadd**
#181 HackAUBG Awards Section

### CODE REVIEW

File: packages/web/src/components/spa/HackAUBG/AwardsSection/AwardsSection.jsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                        id="bronze-img"
+                    ></img>
+                    <p className="award-money" id="bronze-money-amt">
+                        1000 BGN
+                    </p>
+                </div>
+            </div>
+            <div className="awards-section-info-text">
+                <p id="awards-info-text">
+                    And much more!<br></br>
```
Comment (asynchroza) [inline packages/web/src/components/spa/HackAUBG/AwardsSection/AwardsSection.jsx:42]: Text is looking kind of weird, maybe enlarge it a bit and center it? Design peeps should share their opinion here - @annakukova @dankneebee. Furthermore, text size should be consistent with the text in other components whenever we change resolutions. 

`<br></br>` could be written as `<br/>`

File: packages/web/src/components/spa/HackAUBG/AwardsSection/AwardsSection.jsx [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+                        id="silver-img"
+                    ></img>
+                    <p className="award-money" id="silver-money-amt">
+                        2000 BGN
+                    </p>
+                </div>
+                <div className="awards-pic-text">
+                    <img
+                        src="awardIconGold.png"
+                        alt="Award Gold"
```
Comment (asynchroza) [inline packages/web/src/components/spa/HackAUBG/AwardsSection/AwardsSection.jsx:22]: alt tag to be a bit more descriptive

File: packages/web/src/components/spa/HackAUBG/AwardsSection/AwardsSection.jsx [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        <div className="awards-section-hack-aubg-container">
+            <h1 className="awards-section-title">AWARDS</h1>
+            <div className="awards-section-images-container">
+                <div className="awards-pic-text">
+                    <img
+                        src="awardIconSilver.png"
+                        alt="Award Silver"
+                        id="silver-img"
+                    ></img>
+                    <p className="award-money" id="silver-money-amt">
```
Comment (asynchroza) [inline packages/web/src/components/spa/HackAUBG/AwardsSection/AwardsSection.jsx:15]: Can you compress the images?

File: packages/web/src/components/spa/HackAUBG/AwardsSection/awards_section.css [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+.award-pic-text {
+    width: 30vw;
+    height: 30vh;
+    display: flex;
+    flex-direction: column;
+}
+
+#silver-img {
+    max-width: 100%;
+    padding: 14%;
```
Comment (asynchroza) [inline packages/web/src/components/spa/HackAUBG/AwardsSection/awards_section.css:36]: What's the reason behind having 14% here and 18% padding to the other one?

File: packages/web/src/components/spa/HackAUBG/HackAUBG.jsx [hunk 5]
```diff
@@ -7,6 +7,7 @@ import { NavBar } from '../Navigation/NavBar';
 import { Footer } from '../Footer/Footer';
 import { GradingCriteria } from './GradingCriteria/GradingCriteria';
 import { makeBodyScrollable } from '../Navigation/MobileNav/NavMobile';
+import { AwardsSection } from '../HackAUBG/AwardsSection/AwardsSection';
```
Comment (asynchroza) [inline packages/web/src/components/spa/HackAUBG/HackAUBG.jsx:10]: Let's have all imported functions beneath all jsx components. Move `AwardsSection` one row up

**Review by asynchroza — CHANGES_REQUESTED** [review]

File: packages/web/src/components/spa/HackAUBG/AwardsSection/AwardsSection.jsx (hunk 1 above)
Comment (asynchroza) [reply packages/web/src/components/spa/HackAUBG/AwardsSection/AwardsSection.jsx:42]: Reference: 
image

Comment (savinadd) [reply packages/web/src/components/spa/HackAUBG/AwardsSection/AwardsSection.jsx]: done

File: packages/web/src/components/spa/HackAUBG/AwardsSection/awards_section.css [hunk 6]
```diff
@@ -0,0 +1,88 @@
+.awards-section-hack-aubg-container {
+    padding-bottom: 5%;
```
Comment (SuelaCark) [inline packages/web/src/components/spa/HackAUBG/AwardsSection/awards_section.css:2]: Please change all paddings in the css file to margins instead. Good job on using % and em for responsiveness efficiency.

**Review by SuelaCark — CHANGES_REQUESTED** [review]
As mentioned in the comment, change all paddings in the css file to margins instead. While doing so, please check if such change will accidentally effect the items' alignment. 
Everything else looks good. Great job!

File: packages/web/src/components/spa/HackAUBG/AwardsSection/awards_section.css (hunk 6 above)
Comment (savinadd) [reply packages/web/src/components/spa/HackAUBG/AwardsSection/awards_section.css]: ✓

**Review by asynchroza — APPROVED** [review]

**Review by SuelaCark — APPROVED** [review]

**Review by asynchroza — APPROVED** [review]

**Pull request merged**